pkg/controller/managedresource/merger.go

```diff
@@ -29,12 +29,14 @@ const (
 	descriptionAnnotation     = "resources.gardener.cloud/description"
 	descriptionAnnotationText = `DO NOT EDIT - This resource is managed by gardener-resource-manager.
 Any modifications are discarded and the resource is returned to the original state.`
+
+	originAnnotation = "resources.gardener.cloud/origin"
```

Comment:
It should probably be moved to the other annotation constants ([here](https://github.com/gardener/gardener-resource-manager/blob/master/api/resources/v1alpha1/types.go#L22-L33))?

Reply:
the description annotation is also here, both belong together